### DISCUSSION TITLE
Is keycloak not integrating with springboot 3.x framework anymore? I added keycloak to my springboot project today, but it's not working.

### DISCUSSION THREAD

**Discussion by yutianlong520** [keycloak/keycloak#24046] [Feedback]
Is keycloak not integrating with springboot 3.x framework anymore? I added keycloak to my springboot project today, but it's not working.

**Comment by dasniko** [comment]
You should use the Spring Security OAuth stuff out of the box, this will work.
Keycloak adapters are deprecated:
- https://www.keycloak.org/2022/02/adapter-deprecation.html
- https://www.keycloak.org/2023/03/adapter-deprecation-update.html

**Reply by yutianlong520** [reply]
ok, thank you!
